diag_manager/diag_util.F90

```diff
@@ -18,6 +18,7 @@
 !***********************************************************************
 
 MODULE diag_util_mod
+#ifndef use_mpp_io
```

Comment:
Please combine the two as was done in `diag_manager.F90`.

Reply:
This is done with [df04785](https://github.com/NOAA-GFDL/FMS/pull/538/commits/df04785ce6907df1c769a37a5ed03113b55cdf65)